README.md

```diff
@@ -1,6 +1,16 @@
 # EU eHealthNetwork Digitial COVID Certificate value sets
 
-This repository contains the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
+This repository contains a snapshot of the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema). The most up-to-date version of the valuesets are distributed on the DCCG (DCC Gateway).
+
+
+Name | Description
+-----| ------------------
+test-manf | List of recognised manufactorers of tests. Snapshot provided for development purposes, please refer to the gateway for the actual version.
+test-result | List of supported test results (i.e. detected, not detected..)
+test-type | List of supported test types (i.e. PCR, rapid-antigen..)
+disease-agent-targeted | List of diseases/targeted agent 
+country-2-codes | List of ISO-3166-2 country codes
+
```

Comment:
Shouldn't the `vaccine-*.json` files also not be in this table?

Reply:
I agree with adding all the rest files in the table. It helps understand where the visitor will find the information.

We could merger these changes and improve this table in the next step.